osu.Game/Screens/OnlinePlay/DailyChallenge/DailyChallengeIntro.cs

```diff
@@ -83,8 +83,6 @@ public partial class DailyChallengeIntro : OsuScreen
         private SampleChannel? beatmapWindupChannel;
         private SampleChannel? beatmapImpactChannel;
 
-        private IDisposable? duckOperation;
```

Comment:
Rather than nuking, can probably just dispose of this in the `OnSuspending`?

Also did you mean to remove on the `MatchmakingIntroScreen` class?

Reply:
Whoops! Seems okay in `OnSuspending`.